src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -53,46 +53,77 @@ class ThroughputTypes(str, Enum):
+            c.argument('kind', arg_type=get_enum_type(DatabaseAccountKind),
+                       help='The type of Cosmos DB database account to create')
+            c.argument('enable_automatic_failover', arg_type=get_three_state_flag(
+            ), help='Enables automatic failover of the write region in the rare event that the region is unavailable due to an outage. Automatic failover will result in a new write region for the account and is chosen based on the failover priorities configured for the account.')
+            c.argument('capabilities', nargs='+', validator=validate_capabilities,
+                       help='set custom capabilities on the Cosmos DB database account.')
+            c.argument('enable_virtual_network', arg_type=get_three_state_flag(
+            ), help='Enables virtual network on the Cosmos DB database account')
+            c.argument('virtual_network_rules', nargs='+',
+                       validator=validate_virtual_network_rules, help='ACL\'s for virtual network')
+            c.argument('enable_multiple_write_locations', arg_type=get_three_state_flag(
+            ), help="Enable Multiple Write Locations")
+            c.argument('disable_key_based_metadata_write_access', arg_type=get_three_state_flag(
+            ), help="Disable write operations on metadata resources (databases, containers, throughput) via account keys")
+            c.argument('enable_public_network', options_list=[
+                       '--enable-public-network', '-e'], arg_type=get_three_state_flag(), help="Enable or disable public network access to server.")
+            c.argument('enable_analytical_storage', arg_type=get_three_state_flag(
+            ), help="Flag to enable log storage on the account.", is_preview=True)
+            c.argument('network_acl_bypass', arg_type=get_enum_type(NetworkAclBypass), options_list=[
+                       '--network-acl-bypass'], help="Flag to enable or disable Network Acl Bypass.")
+            c.argument('network_acl_bypass_resource_ids', nargs='+', options_list=[
+                       '--network-acl-bypass-resource-ids', '-i'], help="List of Resource Ids to allow Network Acl Bypass.")
+            c.argument('backup_interval', type=int,
+                       help="the frequency(in minutes) with which backups are taken (only for accounts with periodic mode backups)", arg_group='Backup Policy')
+            c.argument('backup_retention', type=int,
+                       help="the time(in hours) for which each backup is retained (only for accounts with periodic mode backups)", arg_group='Backup Policy')
+            c.argument('server_version', arg_type=get_enum_type(
+                ServerVersion), help="Valid only for MongoDB accounts.", is_preview=True)
```

Comment:
It's strange to have `get_enum_type()` seperated to
```
get_enum_type(
)
```
Same as `options_list=[XXX]` to
```
option_list=[
XXX]
```
and `get_three_state_flag()` to
```
get_three_state_flag(
)
```